<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<rosparam command="load" file="$(find scara_ppi_publisher)/config/params.yaml" />
	
	<node name="goal_node"
		  pkg="scara_ppi_publisher"
		  type="goal_publisher.py"
		  args=""
		  output="screen"/>
	<node name="ppi_node"
		  pkg="scara_ppi_publisher"
		  type="control.py"
		  args=""
		  output="screen"/>
	<node name="system_node"
		  pkg="scara_ppi_publisher"
		  type="system.py"
		  args=""
		  output="screen"/>

</launch>
